### PR TITLE
Add Transfer tokens

### PR DESCRIPTION
**Pull request by dnyopr**

### CODE REVIEW

File: src/components/AmountInput.vue [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+  // Set default amount when select another token
+  @Watch("state.plasma.tokenSelected")
+  setDefaultAmount(newVal, oldVal){
+    this.amount = 0
+  }
+
+  validateAmount() {
+    if (this.round) {
+      this.amount = Math.floor(this.value)
```
Comment (redben) [inline src/components/AmountInput.vue]: Should just tell the use only round amounts are allowed.

File: src/components/AmountInput.vue [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    this.$emit("input", Number(this.amount))
+  }
+
+  // Set default amount when select another token
+  @Watch("state.plasma.tokenSelected")
+  setDefaultAmount(newVal, oldVal){
+    this.amount = 0
+  }
+
+  validateAmount() {
```
Comment (redben) [inline src/components/AmountInput.vue]: Although we’re using input type number the value return can be a strong and it should be handled

File: src/components/modals/AddTokenModal.vue [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<template>
+    <b-modal>
+        <b-card>
+            <b-input type="text" v-model="receiverAddress" placeholder="Loom Address"></b-input>
```
Comment (redben) [inline src/components/modals/AddTokenModal.vue]: We could add some validation for the hex address

File: src/components/AmountInput.vue (hunk 2 above)
Comment (nutchy) [reply src/components/AmountInput.vue]: strong or string?
should we emit an amount to parent component as integer type or BN type?

**Pull request merged**